src/Accounts/Authentication.ResourceManager/Models/PSAzureSubscription.cs

```diff
@@ -152,6 +154,8 @@ public string CurrentStorageAccountName
             }
         }
 
+        public PSAzureSubscriptionPolicy SubscriptionPolices;
```

Comment:
I suggest write a getter, so that you don't need to assign value to it in EVERY constructors.
```suggestion
        private PSAzureSubscriptionPolicy _subscriptionPolicies;
        public PSAzureSubscriptionPolicy SubscriptionPolices
        {
            get
            {
                if (this._subscriptionPolicies == null) {
                    this._subscriptionPolicies = new PSAzureSubscriptionPolicy(this.GetSubscriptionPolicies());
                }
                return this._subscriptionPolicies;
            }
        }
```